bip-schnorr/reference.py

```diff
@@ -34,7 +34,7 @@ def bytes_from_point(P):
     return (b'\x03' if P[1] & 1 else b'\x02') + bytes_from_int(P[0])
 
 def point_from_bytes(b):
-    if b[0] in [b'\x02', b'\x03']:
+    if b[0:1] in [b'\x02', b'\x03']:
```

Comment:
Doesn't `b[0:1]` return a list, such that you would be testing for `[x] in [y, z]` rather than `x in [y, z]`?

Reply:
depends what b is -- if it's a list, it'll return a smaller list, if it's a string it'll return a smaller string, if it's a bytes it'll return a smaller bytes.

Reply:
Indeed. In the `list` case, this condition will always be `False`, but maybe we don't have to worry about that here, if only bytes are being thrown around.

Reply:
Doesn't seem like a problem given that this function is called `from_bytes`.